### PR TITLE
benchmarks added

### PR DESCRIPTION
**Pull request by shaharuk-yb**
1. Added following benchmarks to ybbench.
  - tpcc
  - yb-sample-apps
  - sysbench
  - ycsb
2. Readme file added with how to use and how to build

**Testing:** Tested by building the docker image and running the benchmarks

**Next steps:** To push the docker image to dockerhub.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,156 @@
...
+* [How to use](#how-to-use)
+* [Build from source code](#build-from-source-code)
+* [YBBENCH In Interactive Mode](#ybbench-in-interactive-mode)
+
+## What is YB-Bench Docker Container?
+This benchmark docker container is a way to easily run various benchmarks on the YugabyteDB.
+
+This includes following benchmarks pre-installed and ready to be used:
+1. [TPC-C](https://github.com/yugabyte/tpcc)
+2. [SYSBENCH](https://github.com/yugabyte/sysbench)
```
Comment (hbhanawat) [inline README.md]: SYSBENCH -> Sysbench

File: README.md [hunk 2]
```diff
@@ -1 +1,156 @@
...
+3. [YCSB](https://github.com/yugabyte/ycsb)
+4. [YB-SAMPLE-APPS](https://github.com/yugabyte/yb-sample-apps)
```
Comment (hbhanawat) [inline README.md]: YB-SAMPLE-APPS -> yb-sample-apps

File: README.md [hunk 3]
```diff
@@ -1 +1,156 @@
...
+  Above run command will print out following help
+  ```shell
+  ******************************************************************************
+  *                                                                            *
+  *                         YugabyteDB Benchmark Help                          *
+  *                                                                            *
+  ******************************************************************************
+  *         This tool is used to run the benchmarks on YugabyteDB              *
+
+     Syntax: ./run [tpccbenchmark/sysbench/yb-sample-apps/ycsb] [params]
```
Comment (hbhanawat) [inline README.md]: [tpccbenchmark/sysbench/yb-sample-apps/ycsb] -> [tpccbenchmark | sysbench | yb-sample-apps | ycsb]

File: README.md [hunk 4]
```diff
@@ -1 +1,156 @@
...
+
+      Benchmark Options:
```
Comment (hbhanawat) [inline README.md]: This section is not helping much. Can be removed

File: README.md [hunk 5]
```diff
@@ -1 +1,156 @@
...
+
+          tpccbenchmark                 to run tpcc benchmark.
+          sysbench                      to run sysbench benchmark
+          yb-sample-apps                to run yb-sample-apps
+          ycsb                          to run ycsb benchmark
+
+  ******************************************************************************
+  ```
+- #### Get specific benchmark help(-h/--help)
```
Comment (hbhanawat) [inline README.md]: Get specific benchmark help -> Benchmark Help

File: README.md [hunk 6]
```diff
@@ -1 +1,156 @@
...
+   ```shell
+    docker run --name ybbench --rm -it yugabytedb/ybbench:latest ./run <benchmark> --help
+    # example:
+    # docker run --name ybbench --rm -it yugabytedb/ybbench:latest ./run tpccbenchmark -h
+   ```
+  
+- #### Run specific Benchmark
```
Comment (hbhanawat) [inline README.md:54]: Run specific Benchmark -> Run Benchmark

File: README.md [hunk 7]
```diff
@@ -1 +1,156 @@
...
+  - ##### Run sample tpcc workload(Provide comma separated list of nodes). More info can be found on [TPCC](https://github.com/yugabyte/tpcc) repository.
```
Comment (hbhanawat) [inline README.md]: Run sample tpcc workload(Provide comma separated list of nodes). More info can be found on [TPCC](https://github.com/yugabyte/tpcc) repository.  -----> 

(title)Running TPC-C 

(In next line) Info about TPC-C specific parameters can be found [here](https://github.com/yugabyte/tpcc)


Do same for the other benchmarks

File: README.md [hunk 8]
```diff
@@ -1 +1,156 @@
...
+  
+    Alternatively, you can run ybbench in [interactive mode](#ybbench-in-interactive-mode)
+  
+  - ##### Run yb-sample-apps workload. More info can be found on [YB-SAMPLE-APPS](https://github.com/yugabyte/yb-sample-apps) repository.
+    ```shell
+    docker run --name ybbench --rm -it yugabytedb/ybbench:latest ./run yb-sample-apps --workload CassandraKeyValue --value_size 16 --num_unique_keys 1000000 --num_threads_read 0 --num_threads_write 256 --nodes <node1-ip>:9042,<node2-ip>:9042,<node3-ip>:9042 --use_ascii_values create_table_name PerfTest_0 --output_json_metrics 
+    ```
+## Build from source code
+You can build the docker image from the specific branches of respective benchmark repositories.
+1. Clone this repository
```
Comment (hbhanawat) [inline README.md]: Clone this repository ---> Clone Repository

File: README.md [hunk 9]
```diff
@@ -1 +1,156 @@
...
+    ```shell
+    git clone https://github.com/yugabyte/ybbench.git
+    ```
+2. change dir and run docker build
```
Comment (hbhanawat) [inline README.md]: change dir and run docker build -----> Run Docker Build

File: README.md [hunk 10]
```diff
@@ -1 +1,156 @@
...
+    ```shell
+    cd ybbench
+    # master branches is used by default if not specified
+    docker build -t ybbench:0.1 .
+    ```
+    To build from specific branch
```
Comment (hbhanawat) [inline README.md]: To build from specific branch ----> Docker Build with Specific Branch

File: README.md [hunk 11]
```diff
@@ -1 +1,156 @@
...
+    ```shell
+    docker build -t ybbench:0.1 --build-arg tpcc_branch=<my-branch> \
+    --build-arg sysbench_branch=<my-branch> --build-arg sample_apps_branch=<my-branch> \
+    --build-arg ycsb_branch=<my-branch> .
+    ```
+
+## YBBENCH in interactive mode
```
Comment (hbhanawat) [inline README.md]: YBBench ---> ybbench

File: README.md [hunk 12]
```diff
@@ -1 +1,156 @@
...
+Before running any benchmarks you may want to modify/override certain properties files like:
```
Comment (hbhanawat) [inline README.md]: This statement will confuse the users. Should they mount workload.xml or should they run in interactive mode? You should just state that "You can also run ybbench using docker's interactive mode." 

No need to specify workload_all and db.properties. That you have already specified.

**Review by hbhanawat — CHANGES_REQUESTED** [review]
I would like to review it once more after the review comments are addressed.

File: README.md [hunk 13]
```diff
@@ -1 +1,153 @@
...
+    docker build -t ybbench:0.1 .
+    ```
+    Docker Build with Specific Branch
+    ```shell
+    docker build -t ybbench:0.1 --build-arg tpcc_branch=<my-branch> \
+    --build-arg sysbench_branch=<my-branch> --build-arg sample_apps_branch=<my-branch> \
+    --build-arg ycsb_branch=<my-branch> .
+    ```
+
+## ybbench in interactive mode
```
Comment (hbhanawat) [inline README.md:116]: can you move this section above the "Build from source" section

File: README.md [hunk 14]
```diff
@@ -1 +1,153 @@
...
+    ```    
+  
+    Alternatively, you can run ybbench in [interactive mode](#ybbench-in-interactive-mode)
+  
+  - ##### Running yb-sample-apps
+     Info about yb-sample-apps parameters can be found [here](https://github.com/yugabyte/yb-sample-apps).
+    ```shell
+    docker run --name ybbench --rm -it yugabytedb/ybbench:latest ./run yb-sample-apps --workload CassandraKeyValue --value_size 16 --num_unique_keys 1000000 --num_threads_read 0 --num_threads_write 256 --nodes <node1-ip>:9042,<node2-ip>:9042,<node3-ip>:9042 --use_ascii_values create_table_name PerfTest_0 --output_json_metrics 
+    ```
+## Build from source code
```
Comment (hbhanawat) [inline README.md]: Build from source code --> Build from source

**Review by hbhanawat — APPROVED** [review]
Minor comments.

File: README.md (hunk 13 above)
Comment (shaharuk-yb) [reply README.md:116]: done

File: README.md (hunk 14 above)
Comment (shaharuk-yb) [reply README.md]: done

**Pull request merged**